tools/exp/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/aiven-go-client/tools/exp/gen"
+	"github.com/aiven/aiven-go-client/tools/exp/util"
+	"golang.org/x/net/context"
+)
+
+var (
+	// errSettingUpOutputDir is the error returned when the output directory could not be set up.
```

Comment:
I'm not sure why we need all these strings here as package-level vars. They are not adding values, just more noise to the code. Can we use embedded messages for error and logs?

Reply:
I dropped all the constants that are only used once, however the practice to keep repeating stuff in constant is a good practice in general